Lushu/src/main/kotlin/lushu/ContextGrammar/Grammar/Node.kt

```diff
@@ -0,0 +1,95 @@
+package lushu.ContextGrammar.Grammar
+
+class Node(
+    private val regex: String = "",
+    private val sensitive: Boolean = false,
+    private val star: Boolean = false,
+    private val terminal: Boolean = false,
```

Comment:
Just a reminder:  as we discussed in our meeting, `terminal` should probably be renamed to something else.